src/leiningen/i18n.clj

```diff
@@ -79,6 +82,20 @@
   (let [root (:root project)]
     (io/as-file (apply path-join (cons root rest)))))
 
+(defn update-scripts-with-pot-name
+  "CI script task needs to know the name of the POT file to be modified and
+   committed. This will insert the correct name of the POT in the /bin files."
+  [project]
+  (let [dest (path-join (dev-resources-dir project) "i18n" "bin" "update-pot.sh")
+        pot-script (io/resource "leiningen/i18n/bin/update-pot.sh")]
+    (spit (io/as-file dest)
+          (clojure.string/replace-first
+               (slurp pot-script)
+               ; use RegExp's m flag - (?m) - so that ^/$ match start/end of line rather than
+               ; start/end of the entire string
+               #"(?m)^POT_NAME=.*$"
+               (str "POT_NAME=" (:name project) ".pot")))))
```

Comment:
I think this code and the other single-line replacements (starting at line no. 57) would be easier to understand if the implementation detail of needing to specify the multi-line regex flag were hidden in a utility function that provided a way to replace a line in a multi-line string by providing a line regex and the replacement string and having the utility function take care of the details (you can use the regex Pattern's .toString method to get the form that can be fed to `re-pattern` prepended with flag bit to turn the regex supplied by the caller into a multi-line aware regex).